docs/source/install.rst

```diff
@@ -29,17 +29,21 @@ We recommend installing ``repo2docker`` with the ``pip`` tool::
 
     python3 -m pip install jupyter-repo2docker
 
+for the latest release. To install the most recent code from the upstream repository, run::
+
+    python3 -m pip install git+https://github.com/jupyter/repo2docker.git
```

Comment:
```suggestion
    python3 -m pip install https://github.com/jupyter/repo2docker/archive/master.zip
```
Would the suggestion work as well? That way users wouldn't need to have git installed and don't have to transfer the history of the project, just the files that are currently in `master`. I use this form a lot instead of `git+https://` but wondering if there are edge cases?

Reply:
good point, yes that also works